<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOverview</class>
 <widget class="QDialog" name="GeneralOverview">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General Overview</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4" rowspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>76</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="generalPhoto">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>145</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../image/generals/card/caocao.jpg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="changeHeroSkinButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>changeHeroSkin</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="designerLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Designer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="designerLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="cvLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>CV</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="cvLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="illustratorLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Illustrator</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="illustratorLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="companionsLayout">
                <item>
                 <widget class="QLabel" name="companionsLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Companions</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="companionsLineEdit">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="packageLayout">
                <item>
                 <widget class="QLabel" name="packageLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Package</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="packageLineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="returnButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Return...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Search...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="5" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>General Overview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skillLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Skill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="skillTextEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
